lightning/src/ln/channel.rs

```diff
@@ -3502,11 +3557,19 @@ impl<Signer: Sign> Channel<Signer> {
 		cmp::max(self.config.cltv_expiry_delta, MIN_CLTV_EXPIRY_DELTA)
 	}
 
+	pub fn get_max_balance_dust_htlc_msat(&self) -> u64 {
+		self.config.max_balance_dust_htlc_msat
+	}
+
 	#[cfg(test)]
 	pub fn get_feerate(&self) -> u32 {
 		self.feerate_per_kw
 	}
 
+	pub fn get_dust_buffer_feerate(&self) -> u32 {
```

Comment:
Note that this needs to be moved into the previous commit (from "Enforce `max_balance_dust_htlc_msat`..." to "Modify pending inbound/outbound...").